Resources/Prototypes/Maps/core.yml

```diff
@@ -0,0 +1,66 @@
+- type: gameMap
+  id: Core
+  mapName: 'Core'
+  mapPath: /Maps/core.yml
+  minPlayers: 35
+  maxPlayers: 70
+  stations:
+    Core:
+      stationProto: StandardNanotrasenStation
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Core {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationEmergencyShuttle
+          emergencyShuttlePath: /Maps/Shuttles/DeltaV/NTES_UCLB.yml
+        - type: StationJobs
+          overflowJobs:
+            - Passenger
+          availableJobs:
+            # Service
+            Bartender: [ 2, 2 ]
+            Botanist: [ 2, 2]
+            Chef: [ 1, 1 ]
+            Janitor: [ 2, 2 ]
+            Captain: [ 1, 1 ]
+            HeadOfPersonnel: [ 1, 1 ]
+            ServiceWorker: [ 3, 3 ]
+            # Engineering
+            ChiefEngineer: [ 1, 1 ]
+            StationEngineer: [ 4, 4 ]
+            AtmosphericTechnician: [ 2, 2 ]
+            TechnicalAssistant: [ 2, 2 ]
+            # Medical
+            ChiefMedicalOfficer: [ 1, 1 ]
+            MedicalDoctor: [ 3, 4 ]
+            Chemist: [ 2, 2 ]
+            MedicalIntern: [ 2, 2 ]
+            Paramedic: [ 1, 2 ]
+            # Epistemics
+            ResearchDirector: [ 1, 1 ]
+            Scientist: [ 3, 3 ]
+            ResearchAssistant: [ 1, 1 ]
+            Borg: [ 1, 2 ]
+            Chaplain: [ 1, 1 ]
+            Librarian: [ 1, 1 ]
+            ForensicMantis: [ 1, 1 ]
+            # Security
+            HeadOfSecurity: [ 1, 1 ]
+            SecurityOfficer: [ 4, 4 ]
+            Warden: [ 1, 1 ]
+            Lawyer: [ 1, 1 ]
+            SecurityCadet: [ 1, 1 ]
+            Detective: [ 1, 1 ]
+            Prisoner: [ 2, 2 ]
+            # Logistics
+            CargoTechnician: [ 3, 3 ]
+            Quartermaster: [ 1, 1 ]
+            SalvageSpecialist: [ 2, 2 ]
+            # Civilian
+            Clown: [ 1, 1 ]
+            Mime: [ 1, 1 ]
+            Musician: [ 1, 1 ]
+            Boxer: [ 2, 2 ]
+            Reporter: [ 2, 2 ]
```

Comment:
I know this is the same config as Wizden's Core but this is stupid, starts at 35 and doesn't open any more job slots even if it grows to 75 or higher.

Some of these should have some spare Latejoin slots to account for people not cryoing and SSDing or just general pop growth

Reply:
I uhhh... Actually don't normally mess with files like this, so I have no idea how the job slot things work lol. Are you saying the 2nd number is additional slots that open up as the pop increases?

Reply:
Kinda,
first number is round-start.
second number is latejoin slots
-1 is infinite

Reply:
Well, you probably know this a lot better than me. Do you mind suggesting what the numbers would be better off being?